instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+    1.  If the discussion leader feels the trainee is unprepared,
+        she may be asked to do some more work and try again.
+1.  **Final demonstration lessons.**
+    1.  The trainee then does a short demonstration lesson via screen sharing.
+    1.  If the examiner feels that she needs to do more work,
+        she will be given feedback and asked to try again.
+    1.  Trainees do not have to qualify separately for different topics within one Carpentry,
+        but *do* have to do a separate demonstration lesson for each Carpentry.
+1.  **Welcoming to the community.**
+    1.  Once the trainee has passed the demonstration lesson,
+        we generate a PDF certificate for the trainee by running a command-line script in
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
+1.  We keep a list of trainees in a Google Docs spreadsheet with the following fields,
+    which we upate manually:
+    1.  name
+    1.  contact email
+    1.  which training course they took part in
+    1.  the name of the Software Carpentry lesson they're concentrating on (if any)
+    1.  the name of the Data Carpentry lesson they're concentrating on (if any)
+    1.  the URL of their Software Carpentry exercise submission (if any)
+    1.  whether or not they have submitted something for Data Carpentry
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
+	*   we do not record both a Software Carpentry and a Data Carpentry discussion session (though we should)
+    1.  the date on which they passed their Software Carpentry lesson demo (if any)
+	*   this date becomes the award date of the PDF certificate
+        *   we do not currently record the name of the person who was their instructor, or the name of the person who did their final test
+    1.  the date on which they passed their Data Carpentry lesson demo (if any)
+        *   the same caveats apply
+1.  Mentors offer lesson discussion sessions,
+    and trainees sign up for them,
+    on an Etherpad.
+1.  Trainers offer checkout sessions on a second Etherpad.
```

Comment:
1. People who are training to become instructor trainers are called 
"trainees"
2. As part of their training, they must take part in an hour-long 
question-and-answer session about our lessons that is moderated by an 
experienced instructor.  These are called "discussion sessions".
3. After they have done that, they must take part in a demonstration 
session, where they teach for 5 minutes online in front of an audience.  
These are sometimes called "checkout sessions", since we are checking 
them out.